README.md

```diff
@@ -27,8 +27,6 @@ After you're finished please remove all the comments and instructions!
   <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
   <br/>
 
-  <h3><b>Microverse README Template</b></h3>
-
 </div>
 
 <!-- TABLE OF CONTENTS -->
```

Comment:
Please remove the unused sections from the table content

Reply:
Thank you very much for review my PR! I just update the README.md 👍